microsoft/onefuzz
# Can create ado notifications

## Summary of the Pull Request

_What is this about?_

Ports #2435 to C# and makes ADO notifications work. Makes progress towards #1754 

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

1. Submit a new job
2. Create a new ADO template based on the reports container for the job you just created
3. Check that work items are being created
